resources/leiningen/new/kraken_works/project.clj

```diff
@@ -3,21 +3,20 @@
   :url "http://example.com/FIXME"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/tools.logging "0.3.1"]
+  :dependencies [[org.clojure/clojure "1.9.0"]
+                 [org.clojure/tools.logging "0.4.0"]
                  [turbovote.resource-config "0.2.1"]
                  [democracyworks/datomic-toolbox "2.0.4"
                   :exclusions [com.datomic/datomic-pro]]
-                 [prismatic/schema "1.1.6"]
-                 [com.datomic/datomic-pro "0.9.5561"
+                 [com.datomic/datomic-pro "0.9.5661"
                   :exclusions [org.slf4j/slf4j-nop
                                org.slf4j/slf4j-log4j12]]
-                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.128"
+                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.261"
                   :exclusions [commons-codec commons-logging]]
                  [ch.qos.logback/logback-classic "1.2.2"]
-                 [democracyworks/kehaar "0.11.2"]]
+                 [democracyworks/kehaar "0.11.3"]]
```

Comment:
Version 0.11.4 of kehaar was released on Jan. 26. Can we make sure all these deps are the latest (compatible) releases?